Use GFP_ATOMIC instead of GFP_KERNEL since interrupts are disabled.

diff --git a/net/core/port.c b/net/core/port.c
index 0fa33bd..6fea48f 100644
--- a/net/core/port.c
+++ b/net/core/port.c
@@ -69,7 +69,7 @@ static void __port_cache_init(int ifindex)
 		return;
 	}
 
-	port = kzalloc(sizeof(*port), GFP_KERNEL);
+	port = kzalloc(sizeof(*port), GFP_ATOMIC);
 	if (!port) {
 		spin_unlock_irqrestore(&port_cache_lock, flags);
 		return;
@@ -177,7 +177,7 @@ static void port_cache_set_stat_strings(int ifindex, int count, u8 *strings)
 		return;
 	}
 
-	new_strings = kmalloc(count * ETH_GSTRING_LEN, GFP_KERNEL);
+	new_strings = kmalloc(count * ETH_GSTRING_LEN, GFP_ATOMIC);
 	if (!new_strings) {
 		spin_unlock_irqrestore(&port_cache_lock, flags);
 		return;
